src/js_test/test_humidity.js

```diff
@@ -39,25 +35,25 @@ errorSocket.on('message', function(error_message) {
 var configSocket = zmq.socket('push')
 configSocket.connect('tcp://' + creator_ip + ':' + creator_humidity_base_port)
 
-var driverConfigProto = new matrixMalosBuilder.DriverConfig
-// 2 seconds between updates.
-driverConfigProto.delay_between_updates = 2.0
-// Stop sending updates 6 seconds after pings.
-driverConfigProto.timeout_after_last_ping = 6.0
-var hum_params_msg = new matrixMalosBuilder.HumidityParams
-// Real current temperature [Celsius] for calibration 
-hum_params_msg.current_temperature = 23
-driverConfigProto.set_humidity(hum_params_msg)
+var config = matrix_io.malos.v1.driver.DriverConfig.create({
+  delayBetweenUpdates: 2.0,  // 2 seconds between updates
+  timeoutAfterLastPing: 6.0, // Stop sending updates 6 seconds after pings.
+  humidity: matrix_io.malos.v1.sense.HumidityParams.create({
+    currentTemperature: 23   // Real current temperature [Celsius] for calibration 
```

Comment:
Mmm. Better to leave things as they are. We need errors first so we get them if the config goes wrong.